egs2/snips/asr1/run.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set=train
+valid_set=dev
+test_sets="dev test"
+
+asr_config=conf/train_asr_conformer.yaml
+inference_config=conf/decode_asr_rnn.yaml
+
+
+# speed perturbation related
+# (train_set will be "${train_set}_sp" if speed_perturb_factors is specified)
+speed_perturb_factors="0.9 1.0 1.1"
+
+./asr.sh                                               \
+    --use_lm false                                     \
+    --nlsyms_txt  "data/non_linguistic_symbols.txt"    \
+    --audio_format wav                                 \
+    --feats_type raw                                   \
+    --token_type char                                  \
```

Comment:
Hi you should also try with word level decoding since that would be closer to modelling it as classification approach and also has given better performance in other SLU and Speech Dialogue Act benchmark

Reply:
For intent classification, e.g. intent INCREASEBRIGHTNESS , when you say word level decoding, you mean increase brightness (two words)? For sentence level decoding, it means treat the whole INCREASEBRIGHTNESS as one single id. Am I correct? So, what kind of decoding I should try first in your opinion?

Reply:
I would actually suggest trying the whole INCREASEBRIGHTNESS as one single id since it more closely resembles a classification task. We can also try with word level decoding later.